lib/rubycritic/platforms/darwin.rb

```diff
@@ -0,0 +1,26 @@
+require "rubycritic/platforms/base"
+
+module Rubycritic
+  module Platforms
+    class Darwin < Base
+      BROWSER_MAP = {}.tap do |hash|
+        hash.merge! :chrome => "Google Chrome", :firefox => "firefox", :safari => "safari"
```

Comment:
@Onumis  yes, i test it in my mac osx 10.10.5.
 [http://superuser.com/questions/350309/how-can-i-open-a-url-in-google-chrome-from-the-terminal-in-os-x](http://superuser.com/questions/350309/how-can-i-open-a-url-in-google-chrome-from-the-terminal-in-os-x)